backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/ScheduledPublicationService.java

```diff
@@ -0,0 +1,75 @@
+package de.bund.digitalservice.ris.caselaw.domain;
+
+import java.time.LocalDateTime;
+import lombok.extern.slf4j.Slf4j;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class ScheduledPublicationService {
+
+  private final DocumentationUnitRepository repository;
+
+  private final HandoverService handoverService;
+
+  /**
+   * "Terminierte Abgabe": Cron job that finds doc units that have a scheduled publishing date-time
+   * now, or in the past and published them. Even if publication fails, we will unset the scheduling
```

Comment:
I don't understand the grammar. Is this correct?
```suggestion
   * now, or in the past and publishes them. Even if publication fails, we will unset the scheduling
```